gm4_orb_of_ankou/soul_essence.csv

```diff
@@ -0,0 +1,182 @@
+-Run beet to update columns 4-9

```

Comment:
I had no idea you could add comments to csvs

Reply:
It's a normal cell actually, I just skip the first few lines when parsing it then print that out when rewriting it